### PR TITLE
CHK-1061: Remove broken tests and update documentation

### PR DESCRIPTION
**Pull request by wdsrocha**
#### What is the purpose of this pull request?

1. Remove all remaining broken tests (that is, tests that never pass)
2. Update documentation mostly by removing unused or outdated parts

#### What problem is this solving?

Nowadays when running the entire test suite, it's hard to know which tests failed due to a new feature/bugfix or if it's a test that is broken for a really long time. By fixing all tests, that problem would go away, but not all tests were easy to fix, so we opted to just discard those tests:

- \[Shipping\] Delivery + Scheduled Delivery and Delivery - Credit card (vtexgame1, clean, geo, invoice, nolean)
- [Shipping Preview] Delivery + Scheduled Delivery + Pickup (nolean)
- [Shipping Preview] Delivery + Scheduled Delivery + Scheduled Pickup (nolean)
- [Shipping Preview] Delivery Only + Delivery Pickup (nolean)

We could spend some effort to fix them, but there are other known issues with more priority. You can read more details about what is happening in each test [here](https://www.notion.so/vtexhandbook/Ressurrei-o-de-testes-E2E-e1b3fcea081342e2bcd1defedd0dddab).

Besides that, I updated the documentation where I think it was necessary (like removing `SCENARIOS.md`, which was very outdated).

#### How should this be manually tested?

I don't think there is need to be manually test, but you can check [the Cypress Dashboard run for this branch](https://dashboard.cypress.io/projects/kobqo4/runs/5514/) :satisfaction:

### CODE REVIEW

**Review by lucasecdb — APPROVED** [review]

**Comment by wdsrocha** [comment]
**UPDATE**

If you notice the test summary from Cypress above, you'll see that it skips 25 tests :wat:. I won't describe in detail here, but my sieve to find out which tests were disabled/broken failed to find tests that were disabled and now they pass! So I decided to also use this PR to re-enable the ones that I'm sure that are working fine. This last commit will only clean 10 tests, but the other 15 have a catch (they only fail on Saturday during the dawn), so I'm still investigating what to do with em.

Also, I used this PR to re-trigger the Jenkins build, which failed due to a Docker rate limit 😬

**Review by lucasecdb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -113,3 +113,16 @@ The main purpose of this repository build a more effective e2e test suite for th
...
+In the end of 2021 we realized that some all-time-broken tests were impacting our confiability, so we decided to fix them. Unfortunately some of them represented real bugs with Checkout, but after finding out how rare and hard to fix those scenarios were, some investigation, we opted to just discard those tests.
+
+Those tests were:
+
+- \[Shipping\] Delivery + Scheduled Delivery and Delivery - Credit card (vtexgame1, clean, geo, invoice, nolean)
+- [Shipping Preview] Delivery + Scheduled Delivery + Pickup (vtexgame1nolean)
+- [Shipping Preview] Delivery + Scheduled Delivery + Scheduled Pickup (vtexgame1nolean)
+- [Shipping Preview] Delivery Only + Delivery Pickup (vtexgame1nolean)
+
+You can check them using version control but additional information can be found [here](https://www.notion.so/vtexhandbook/Ressurrei-o-de-testes-E2E-e1b3fcea081342e2bcd1defedd0dddab).
```
Comment (lbebber) [inline README.md:128]: boa

Comment (lbebber) [reply README.md:128]: talvez um link pro commit atualmente no topo do master tambem seja util

**Review by lbebber — APPROVED** [review]
just 1 suggestion but LGTM

File: README.md (hunk 1 above)
Comment (wdsrocha) [reply README.md]: boa, adicionei

**Comment by wdsrocha** [comment]
merging and releasing next week, due to docker pull rate limit on jenkins (it resets in 6 hours cycles)

**Review by lucasecdb — APPROVED** [review]

**Review by lbebber — APPROVED** [review]

**Pull request merged**